test/quantization/fx/test_quantize_fx.py

```diff
@@ -3511,6 +3511,44 @@ def forward(self, x):
                 expected_node_occurrence=node_occurrence,
                 expected_node_list=node_list)
 
+    def test_stack_trace_preserved(self):
+        class M(nn.Module):
+            def __init__(self):
+                super().__init__()
+                self.linear = nn.Linear(1, 1)
```

Comment:
could we check if the stack trace is preserved for lowering, we can do that by testing quantization for a single functional relu

Reply:
sure, planning to do in a future PR, hope that's ok